packages/next-swc/crates/next-core/src/next_config.rs

```diff
@@ -420,6 +423,7 @@ enum MiddlewarePrefetchType {
 pub struct CompilerConfig {
     pub react_remove_properties: Option<bool>,
     pub relay: Option<RelayConfig>,
+    pub emotion: Option<serde_json::Value>,
```

Comment:
You could let serde do the conversion.

```
#[serde(untagged)]
enum EmotionTransformOptionsOrBoolean {
  Boolean(bool),
  Options(EmotionTransformOptions)
}
```

But that breaks with the `EmotionTransformOptions::enabled` flag, which is weird anyway. Why having a enabled flag instead or (or even in addition to) `Option<EmotionTransformOptions>`. I think that should be removed.